python/spacewalk/common/rhnLog.py

```diff
@@ -223,6 +224,8 @@ def logMessage(self, *args):
         msg = "%s%s.%s" % (self.log_info, module, tbStack[callid][2])
         if args:
             msg = "%s%s" % (msg, repr(args))
+        if self.component:
+            msg = "%s%s" % (self.component, msg)
```

Comment:
Just to be sure this does have some spacing between component and actual message, right?

Just from reading the code I see `msg` starts with ip address, but I am not sure if `component` has a space at the end.